.travis.yml

```diff
@@ -8,8 +8,8 @@ python:
 sudo: false
 
 install:
-  - pip install --quiet --requirement requirements.txt
-  - pip install --quiet pylint
+  - pip install -r requirements.txt
+  - pip install pylint
```

Comment:
1. Travis automatically collapses the output anyways.
2. True, I'll change that.